src/controllers/__tests__/column.test.ts

```diff
@@ -40,9 +40,11 @@ describe("updateColumnsSort", () => {
         sort: 0,
       },
     });
+    const parsedBody = JSON.parse(res.body);
 
     expect(res.statusCode).toEqual(200);
-    expect(res.body).toContain(mockProject.id);
+    expect(parsedBody.id).toBe(mockProject.id);
+    // TODO: more test
```

Comment:
Random but this comment was funny to me for some reason

Reply:
Oops, it was my internal before making PR... haha